corehq/apps/hqwebapp/management/commands/resource_static.py

```diff
@@ -37,11 +37,12 @@ def current_sha(self):
         return sha
 
     def output_resources(self, resources, overwrite=True):
-        mode = 'w+' if overwrite else 'w'
-        from get_resource_versions import get_resource_versions
-        resource_versions = get_resource_versions()
-        resource_versions.update(resources)
-        with open(os.path.join(self.root_dir, 'resource_versions.yaml'), mode) as fout:
+        resource_versions = {}
+        if not overwrite:
+            from get_resource_versions import get_resource_versions
+            resource_versions = get_resource_versions()
+            resource_versions.update(resources)
```

Comment:
I think there needs to be an `else` clause here. Otherwise `resource_versions` will be empty (when `overwrite == True`), which I assume is not desired. What do you think of adding some unit tests for this method?

Reply:
Fixed in https://github.com/dimagi/commcare-hq/pull/24938/commits/26c520d94b417a32569699e29e61e81bcc115418 and added tests in https://github.com/dimagi/commcare-hq/pull/24938/commits/71f56d5a62e1f22d977873f24e4b3da06fc4843a